.github/workflows/build.yml

```diff
@@ -110,7 +110,7 @@ jobs:
     - name: Create docker tag
       id: create_docker_tag
       run: |
-        TAG=sbtscala/scala-sbt:${{ matrix.javaTag }}_${{ steps.get_sbt_version.outputs.VERSION }}_${{ matrix.scalaVersion }}
+        TAG=carbrex/scala-sbt:${{ matrix.javaTag }}_${{ steps.get_sbt_version.outputs.VERSION }}_${{ matrix.scalaVersion }}
```

Comment:
Euh, what is the idea here?

Reply:
I changed this so these tests can run on my repository. I have changed it back to sbtscala.